internal/hardware/retrieve.go

```diff
@@ -44,9 +44,9 @@ func (c *Client) Retrieve() *cobra.Command {
 			projectID, _ := cmd.Flags().GetString("project-id")
 			hardwareReservationID, _ := cmd.Flags().GetString("id")
 
-			header := []string{"ID", "Facility", "Plan", "Created"}
+			header := []string{"ID", "Facility", "Metro", "Plan", "Created"}
 
-			inc := []string{}
+			inc := []string{"facility"}
```

Comment:
Is it included in standard cases too?

Reply:
Yes, updated the switch case to consider `facility.metro` only as we only need that much.